rallyrolebot/constants.py

```diff
@@ -39,3 +40,17 @@
 ERROR_COLOR =  Color(0xFF0000)
 SUCCESS_COLOR = Color(0x0000FF)
 WARNING_COLOR = Color(0xFFFF00)
+WHITE_COLOR = Color(0xFFFFFE)
+INCREASE_DECREASE_GRADIENT_COLOR = [
+    Color(0x9E1711), # Spartan Crimson
+    Color(0xB12E21), # Chinese Brown
+    Color(0xC34632), # International Orange
+    Color(0xD65D42), # Medium Vermilion
+    Color(0xE97452), # Burnt Sienna
+    WHITE_COLOR,
+    Color(0x95F985), # Light Green
+    Color(0x4DED30), # Neon Green
+    Color(0x26D701), # Harlequin Green
+    Color(0x00C301), # Vivid Malachite
+    Color(0x00AB08), # Islamic Green
+]
```

Comment:
More of a TODO than a suggestion - we could create an interpolation function that uses math to calculate the gradient.